app/page.js

```diff
@@ -25,6 +32,7 @@ export default async function Home() {
   posts.sort((a, b) => {
     return Date.parse(a.date) < Date.parse(b.date) ? 1 : -1;
   });
+  await generateFeed(metadata, posts);
```

Comment:
This feels kinda hacky. Can we use a [route handler](https://nextjs.org/docs/app/api-reference/file-conventions/route) instead? Does that work with static export?

Reply:
https://nextjs.org/docs/app/building-your-application/routing/route-handlers#non-ui-responses

Reply:
Yes, you're right. I've updated the PR to use route handlers instead. It still seems to work fine with static exports.